tests/datatypes/test_int.py

```diff
@@ -38,34 +38,37 @@ class BinaryIntOperationTests(BinaryOperationTestCase, TranspileTestCase):
 
     not_implemented = [
         'test_modulo_complex',
-
-        'test_multiply_complex',
-
-        'test_power_complex',
-
-        'test_true_divide_complex',
     ]
 
 
 class InplaceIntOperationTests(InplaceOperationTestCase, TranspileTestCase):
     data_type = 'int'
 
-    not_implemented = [
-
-
-        'test_modulo_complex',
-        'test_modulo_float',
-
-        'test_multiply_bytearray',
-        'test_multiply_bytes',
-        'test_multiply_complex',
-        'test_multiply_float',
-        'test_multiply_list',
-        'test_multiply_tuple',
-
-        'test_power_complex',
-
-
-
-        'test_true_divide_complex',
-    ]
+    substitutions = {
```

Comment:
I'm a little surprised this is needed; `tests/utils.py` should normalize floating point values so that differences in extreme decimal point values shouldn't matter. I'll merge this as is because it's an improvement on what was there before, but if you get a chance, it would be worth investigating to see why this isn't happening.